client/utils.go

```diff
@@ -69,3 +71,9 @@ func ReadPageRequest(flagSet *pflag.FlagSet) (*query.PageRequest, error) {
 		Reverse:    reverse,
 	}, nil
 }
+
+// NewClientFromNode sets up Client implementation that communicates with a Tendermint node over
+// JSON RPC and WebSockets
+func NewClientFromNode(nodeURI string) (*rpchttp.HTTP, error) {
+	return rpchttp.New(nodeURI, "/websocket")
+}
```

Comment:
related to https://github.com/cosmos/cosmos-sdk/pull/8953#discussion_r599680243. I understand this is what we currently do in the SDK for the `--node` flag, and it's maybe not the concern of this PR.

Should we create an issue for this?
- https://docs.tendermint.com/master/rpc/ says the RPC endpoint is without `/websocket`
- is it alright to use a HTTP client on the websocket endpoint?

Reply:
I have opened an [issue](https://github.com/cosmos/cosmos-sdk/issues/8986) to address this question